### PR TITLE
Initial project configuration

### PR DESCRIPTION
**Pull request by fedefernandez**
Could you please review @juanpedromoreno? Thanks!

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
+import catext.Dependencies._
+
+val dev  = Seq(Dev("47 Degrees (twitter: @47deg)", "47 Degrees"))
+val gh   = GitHubSettings("com.fortysevendeg", "github4s", "47 Degrees", apache)
```
Comment (juanpedromoreno) [inline build.sbt]: github4s?

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+lazy val pluginSettings = Seq(
+    sbtPlugin := true,
+    scalaVersion in ThisBuild := "2.10.6",
+    resolvers ++= Seq(
+      Resolver.sonatypeRepo("releases"),
+      "jgit-repo" at "http://download.eclipse.org/jgit/maven"
+    ),
+    libraryDependencies ++= Seq(
+      "com.lihaoyi"   %% "scalatags" % "0.6.0",
```
Comment (juanpedromoreno) [inline build.sbt]: Why do we need scalatags here?

File: project/plugins.sbt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+addSbtPlugin("com.fortysevendeg" % "sbt-catalysts-extras" % "0.1.2")
+addSbtPlugin("com.geirsson"      % "sbt-scalafmt"         % "0.4.7")
```
Comment (juanpedromoreno) [inline project/plugins.sbt]: 0.4.10

**Review by juanpedromoreno — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 2 above)
Comment (fedefernandez) [reply build.sbt]: We can remove it, not needed at all

**Comment by fedefernandez** [comment]
Comments addresses 👍

**Review by juanpedromoreno — APPROVED** [review]

**Pull request merged**